src/components/time.js

```diff
@@ -5,8 +5,8 @@ import Timer from './timer';
 import { TimeContainer } from './time-components';
 
 export default class Time extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
```

Comment:
Should we also pass `context`? I know theming uses it. (Same with other components)

Or alternatively 

```
constructor() {
  super(...arguments);
```